osbs/core.py

```diff
@@ -43,14 +43,17 @@
 logger = logging.getLogger(__name__)
 
 
-def check_response(response):
+def check_response(response, log_errors=True):
     if response.status_code not in (httplib.OK, httplib.CREATED):
         if hasattr(response, 'content'):
             content = response.content
         else:
             content = b''.join(response.iter_lines())
 
-        logger.error("[%d] %s", response.status_code, content)
```

Comment:
well we want to either error or debug, I don't think we need more levels to use,
and the level int for error is 40 and debug 10, isn't very readable